modelwriter.go

```diff
@@ -258,6 +225,40 @@ func (w *modelWriter) buildModelError(out *model.Error, e *ErrorData) {
 	out.Culprit = truncateString(out.Culprit)
 }
 
+// BuildModelTransaction converts apm transaction to model transaction
```

Comment:
I don't love exposing this just for Lambda. `td.Context.build()` mutates the transaction context, for one thing. I think it's okay as it is, but could be a source of bugs later down the line.

Maybe we could have a method which only copies immutable properties (IDs, sampled status, sample rate) to the model object? Or otherwise, perhaps we could expose those properties as methods on Transaction, and build the partial transaction model directly in module/apmlambda code. Would that be awful?